src/pages/membership/components/createYouthProfileForm/CreateYouthProfileForm.tsx

```diff
@@ -126,16 +82,17 @@ function CreateYouthProfileForm(props: Props) {
   const { t } = useTranslation();
   const languages = ['FINNISH', 'SWEDISH', 'ENGLISH'];
 
+  const getAge = () => {
```

Comment:
This function is declared twice in these changes. Might be worth considering moving it to `utils` etc.

Reply:
Removed `getAge` function and age is now saved directly to variable.